src/printapp/templates/index.html

```diff
@@ -272,52 +259,45 @@ <h3 class="subheader">Welcome, {{email}}</h3>
         <a href="#" class="close">&times;</a>
       </div>
       {{#if fileSelected}}
-        <p>
-          <button {{action 'selectFile'}}>Select a file</button>
-        </p>
+		<h5>Drop a file here or</h5>
+		  <p><button {{action 'selectFile'}}>Select a file</button></p>
+		  <p>
+			Tip: PDF files work best.
+		  </p>
```

Comment:
Since drag and drop doesn't actually work, we can't merge this. Unless @tylervz wants to add drag and drop support, I think our best bet is to have UI which doesn't imply that drag and drop is possible.